lisa/features/gpu.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from enum import Enum
+from typing import Any, cast
+
+from lisa.base_tools.wget import Wget
+from lisa.feature import Feature
+from lisa.operating_system import Linux, Redhat, Ubuntu
+from lisa.tools import Uname
+from lisa.util import LisaException
+
+FEATURE_NAME_GPU = "Gpu"
+
+ComputeSDK = Enum(
+    "ComputeSDK",
+    [
+        # GRID Driver
+        "GRID",
+        # CUDA Driver
+        "CUDA",
+    ],
+)
+
+# Link to the latest GRID driver
+# The DIR link is
+# https://download.microsoft.com/download/9/5/c/95c667ff-ab95-4c56-89e0-e13e9a76782d/NVIDIA-Linux-x86_64-460.32.03-grid-azure.run
+GRID_DRIVER = "https://go.microsoft.com/fwlink/?linkid=874272"
+
+
+class Gpu(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = self._node.log
```

Comment:
Refer to Tool, add the log in Feature class like below,
`self._log = get_logger("feature", self.name, self.node.log)`

Reply:
> Refer to Tool, add the log in Feature class like below,
> `self._log = get_logger("feature", self.name, self.node.log)`

Shouldn't be similar to what is there in startstop.py ?

Reply:
Good catch. it should be fixed too. For those small fixes, you can create some small commit, and make changes history more clear.

Reply:
Done